system-tests/fixtures/api-impl-js/ern-movie-api-impl/package.json

```diff
@@ -1,25 +1 @@
-{
-  "name": "ern-movie-api-impl",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT",
-  "devDependencies": {
-    "react-native": "0.60.5"
-  },
-  "dependencies": {
-    "react-native-ernmovie-api": "0.0.11"
-  },
-  "ern": {
-    "containerGen": {
-      "hasConfig": false,
-      "moduleName": "ReactNativeErnmovieApiImplJs",
-      "apiNames": [
-        "Movies"
-      ]
-    },
-    "moduleType": "ern-js-api-impl"
-  },
-  "keywords": [
-    "ern-js-api-impl"
-  ]
-}
+{"name":"ern-movie-api-impl","version":"1.0.0","main":"index.js","license":"MIT","devDependencies":{"react-native":"0.60.6"},"dependencies":{"react-native-ernmovie-api":"0.0.11"},"ern":{"containerGen":{"hasConfig":false,"moduleName":"ReactNativeErnmovieApiImplJs","apiNames":["Movies"]},"moduleType":"ern-js-api-impl"},"keywords":["ern-js-api-impl"]}
```

Comment:
Wait .. Is this expected?!

Reply:
No ;) As mentioned in my PR comment, package.json generation of api impls seems to have been broken somehow in a previous merge. Will open a separate issue for that.

Reply:
Oops sorry, I did read the description but somehow missed that part ;)